docs/compatibility.md

```diff
@@ -24,3 +24,16 @@ With the monkey patch disabled, use `render_component` (or  `render_component_to
 ## Bridgetown (Static Site Generator)
 
 [Bridgetown](https://www.bridgetownrb.com/) supports ViewComponent via an experimental shim provided by the [bridgetown-view-component gem](https://github.com/bridgetownrb/bridgetown-view-component). More information available [here](https://www.bridgetownrb.com/docs/components/ruby#need-compatibility-with-rails-try-viewcomponent-experimental).
+
+## ActionText Support
+
+Trying to use `rich_text_area` via ActionText out of the box with ViewComponent will result in this error:
+```
+undefined method `rich_text_area_tag'
+```
```

Comment:
```suggestion

```
undefined method `rich_text_area_tag'
```
```

Reply:
Hey @joelhawksley, I don't think this suggestion applied correctly due to using ``` inside the suggestion block